kedro_mlflow/io/mlflow_metrics_dataset.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+from pathlib import Path, PurePosixPath
+from typing import Any, Dict, List, Optional, Union
+
+import mlflow
+from cachetools import Cache
+from kedro.io import AbstractVersionedDataSet
+from kedro.io.core import DataSetError, Version
+
+
+class MlflowMetricsDataSet(AbstractVersionedDataSet):
+    """This class represent mlflow metrics.
+
+    It decorates their ``save`` method to log the metrics in mlflow when
```

Comment:
I think single pair of backticks is usually used. This appears in multiple places.

Reply:
I believe this is related to the Sphinx configuration. I used convention used in this project.

Reply:
My bad, I use double backticks because it is what I am used too, but I don't know if it is common/best practice. I don't remember having set any particular settings for Sphinx.

Reply:
I might be thinking about Markdown whereas this is being parsed by ReStructuredText parser. That's a thing to check.